### ISSUE TITLE
Implement faceting on A-Z page

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/ucsc-library-digital-collections#507]
### Descriptive summary
As an end user, I want to find collections based on subject, place, etc. [facets] and keyword [search]

Questions to consider:
- Is your feature request related to a problem? Please describe.
- What would you like to have happen?
- Who or what activities does this feature serve or support?

### Acceptance criteria
**Done means:**
- [ ]
- [ ]

**Comment by JessWaggs** [comment]
Mock up for desktop and mobile
[AtoZ_Page_rev3_facets.pdf](https://github.com/UCSCLibrary/ucsc-library-digital-collections/files/10006536/AtoZ_Page_rev3_facets.pdf)

**Comment by rschwab** [comment]
@JessWaggs this is ready for QA on [sandbox](http://digitalcollections-staging-sandbox.library.ucsc.edu/collections).

**Comment by rschwab** [comment]
That's an easy change, I'll let you know when its in sandbox.

**Comment by rschwab** [comment]
@JessWaggs this is now in sandbox:


![Image](https://user-images.githubusercontent.com/515412/233206725-704ddf68-2382-4f04-ab92-d9da789fd5c9.png)

**Comment by JessWaggs** [comment]
Looks great - thanks Ryan!

**Comment by rschwab** [comment]
Yes, this is the facet panel bug form 386. It should only be searching for matching collections, not matching works. If you click a facet that appears on the page (and not the 'more' popup window) then it works as expected.

**Comment by JessWaggs** [comment]
I reviewed this in desktop and mobile and both look great!
